docs/user/security/securing-communications/index.asciidoc

```diff
@@ -4,127 +4,178 @@
+
+[source,sh]
 --------------------------------------------------------------------------------
-elasticsearch.hosts: ["https://<your_elasticsearch_host>.com:9200"]
+openssl pkcs12 -in elastic-certificates.p12 -cacerts -nokeys -out elasticsearch-ca.crt
 --------------------------------------------------------------------------------
+--
 
-Using the HTTPS protocol results in a default
-`elasticsearch.ssl.verificationMode` option of `full`, which utilizes hostname
-verification.
+. *Configure {kib} to trust the {es} certificates for the HTTP layer.*
++
+You need to specify the information required to access your {es} CA certificate chain.
 
-For more information, see <<settings,{kib} configuration settings>>. 
+.. If your CA certificate chain is in PEM format:
++
+--
+Specify one or more certificates in `kibana.yml`:
+
+[source,yaml]
+--------------------------------------------------------------------------------
+elasticsearch.ssl.certificateAuthorities: ["/path/to/elasticsearch-ca.crt"]
+--------------------------------------------------------------------------------
 --
 
-. Specify the {es} cluster's CA certificate chain in `kibana.yml`:
+
+.. If your CA certificate chain is in a PKCS#12 file:
 +
 --
+WARNING: You should not use a PKCS#12 file that contains a private key. This is an unnecessary security risk. If you only have a PKCS#12
+file that contains a private key, a safer approach is to extract the CA certificate chain in PEM format as described above.
 
-If you are using your own CA to sign certificates for {es}, then you need to
-specify the CA certificate chain in {kib} to properly establish trust in TLS
-connections. If your CA certificate chain is contained in a PKCS #12 trust store,
-specify it like so:
+Specify your PKCS#12 file in `kibana.yml`:
 
 [source,yaml]
 --------------------------------------------------------------------------------
-elasticsearch.ssl.truststore.path: "/path/to/your/truststore.p12"
-elasticsearch.ssl.truststore.password: "optional decryption password"
+elasticsearch.ssl.truststore.path: "/path/to/elasticsearch-ca.p12"
 --------------------------------------------------------------------------------
 
-Otherwise, if your CA certificate chain is in PEM format, specify each
-certificate like so:
+If your PKCS#12 file is encrypted, add the decryption password to your <<secure-settings,{kib} keystore>>:
 
 [source,yaml]
 --------------------------------------------------------------------------------
-elasticsearch.ssl.certificateAuthorities: ["/path/to/your/cacert1.pem", "/path/to/your/cacert2.pem"]
+bin/kibana-keystore add elasticsearch.ssl.truststore.password
+--------------------------------------------------------------------------------
+
+TIP: If your PKCS#12 file isn't protected with a password, depending on how it was generated, you may need to set
+`elasticsearch.ssl.truststore.password` to an empty string.
+--
```

Comment:
I am afraid that some users will attempt to use `elastic-stack-ca.p12` as a trust store. However, I think we still need to mention that `elasticsearch.ssl.truststore.path` is an option here, otherwise users may not be aware of it.

This, with the included warning, is the best compromise I could come up with to discourage this. The warning is sort of vague, but I think it gets the point across.